lib/rubocop/cop/rspec/factory_bot/consistent_parentheses_style.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module FactoryBot
+        # Use a consistent style for parentheses in factory bot calls
+        #
+        # @example
+        #
+        #   # bad
+        #
+        #   create :user
+        #   build(:user)
+        #   create(:login)
+        #   create :login
+        #
+        #   # good
+        #   create :user
+        #   build :user
+        #   create :login
+        #   create :login
+        #
+        #   # good
+        #   create(:user)
+        #   create(:user)
+        #   create(:login)
+        #   build(:login)
+        #
+        class ConsistentParenthesesStyle < Base
+          extend AutoCorrector
+          include ConfigurableEnforcedStyle
+
+          MSG_ENFORCE_PARENS = 'Prefer method call with parentheses'
+          MSG_OMIT_PARENS = 'Prefer method call without parentheses'
+
+          def_node_matcher :factory_call, <<-PATTERN
+            {
+              (send
+                ${(const nil? {:FactoryGirl :FactoryBot}) nil?} :create (sym $_)
+              $...)
+              (send 
+                ${(const nil? {:FactoryGirl :FactoryBot}) nil?} :build (sym $_)
+              $...)
+            }
+          PATTERN
+
+          def on_send(node)
+            return if nested_call?(node)
```

Comment:
But why not? For performance?

Reply:
Because create(:default_country, create :default_country) nesting without parens is not a valid call. Ruby forces you to put parens around the nested factory. So checking it only causes issues

Reply:
Ruby tells me the following: 

`let(:user) { create(:user) }` == valid

`let(:user) { create(:user, create :user) }` == very not valid

For some reason it is expecting a ( in this format

Reply:
Right. Sees there's a difference when there is more than one argument.

However, if it's invalid syntax, why should we care about it in the cop?
```
$ ruby-parse -e 'create(:user, create :user)'
(fragment:0):1:22: error: unexpected token tSYMBOL
(fragment:0):1: create(:user, create :user)
(fragment:0):1:                      ^~~~~
```

Reply:
That is a fair point ^^. But I'd prefer not to iterate on the nested calls because that could cause issues with the cop trying to fix the nested calls.

Reply:
Ha, I finally got it. Sorry for being a 🐌
So this check is to prevent auto-correction for `create(:user, create(:user))` to a nonsense when `omit_parentheses` is the enforced style.
I'm all for having this check 👍 Thanks for standing your ground.

However, due to how `nested_call?` is implemented, it might still attempt to auto-correct the following for example:
```ruby
puts(1, create(:user))
```

To cover this it will be enough to check `node.parent.send_type?`.

Reply:
I changed the implementation and used your example as a test :D and thank you for being so patient during this discussion ^^